Documentation/ceph-block.md

```diff
@@ -228,6 +231,7 @@ parameters:
   clusterNamespace: rook-ceph
   # Specify the filesystem type of the volume. If not specified, it will use `ext4`.
   fstype: xfs
+allowVolumeExpansion: true
```

Comment:
It's probably best to add a `works for Kubernetes 1.14+` note as a reminder to all users.